src/v2/Apps/getAppNovoRoutes.tsx

```diff
@@ -0,0 +1,23 @@
+import { buildAppRoutes } from "v2/Artsy/Router/buildAppRoutes"
+import { RouteConfig } from "found"
+import { routes as artistRoutes } from "v2/Apps/Artist/routes"
+import { debugRoutes } from "./Debug/debugRoutes"
+import { cloneDeep } from "lodash"
+
+export function getAppNovoRoutes(): RouteConfig[] {
+  return buildAppRoutes([
+    {
+      routes: cloneDeep(artistRoutes).map(route => {
```

Comment:
Why are we using `cloneDeep` here?

Reply:
We are using a mutable data structure to define the routes and this map operation alters them for both novo and force. I plan on refactoring the routing to use classes or something a bit more stable.

Reply:
Wont cloneDeep then dupliciate everything within the `Component` key of the tree? Like everything inside of [artistRoutes](https://github.com/artsy/force/blob/master/src/v2/Apps/getAppRoutes.tsx#L22) and its references?

Reply:
(Just something to keep an eye on)

Reply:
This is a fair concern, I didn't think about what all was attached to the route. I expected that it was a simple configuration object, I doubt there would be any issue with deep cloning here aside from potential wasted memory.